src/main/java/aeneas/views/BuildLevelView.java

```diff
@@ -93,5 +97,8 @@ public void initialize(URL location, ResourceBundle resources) {
     centerBox.setAlignment(Pos.TOP_RIGHT);
     centerBox.getChildren().add(boardView);
 
+    this.lookup("#saveButton").setOnMouseClicked(
```

Comment:
Actually, let me check that again, I realized that there may have been a different problem causing that.